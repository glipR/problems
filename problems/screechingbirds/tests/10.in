33 377
59 76 dpgdzszwqrqrxqtqfybuz
49 7 zvhoamnbyaqjlds
80 6 aeaiukjemltsziwlyjdl
28 52 rwwvmt
17 73 naxcetwmalxhcrccdzfqtlkyd
52 100 fqbifcuhxixbyxurfbbqyyb
99 42 vmpiqmwormx
12 15 lkypjuqklahu
98 59 ioxgxyq
47 100 ugqlf
76 26 xqnfpccysyetkjbqt
4 28 ypsynxykslwvktwrxez
62 13 cshosjmdvfnf
20 85 xpxobdwgnpaabesi
11 35 scbaqvofit
12 53 letbgqoipklkcpsu
98 67 ksawzcehoejnob
35 14 hjkkdltwjzkbeu
86 55 cpshruduoizkzu
90 16 omlbfiuvfvfluq
2 8 ejzfeyovxqtqocik
27 16 lzyriieax
29 88 ltztrqdagsccmjvclmgabyalx
0 33 twksexaxxpkiauklmyrzzxhn
70 54 bibnfjpruwcju
27 6 pgecwrltawtlsjq
3 17 nprimartmqkmdxuy
60 69 doxjanznswfrwjcwvvczoga
71 47 ktnnpk
12 88 dhbgdjyijmsiydlbtgnzgg
63 99 nzzvfdbblnkpmhzskz
10 13 pyfghsvavjlqgpmejmsf
97 4 zbkoamuhhgxkovzazh