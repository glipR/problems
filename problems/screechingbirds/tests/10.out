twksexaxxpkiauklmyrzzxhn
ejzfeyovxqtqocik
nprimartmqkmdxuy
ypsynxykslwvktwrxez
ejzfeyovxqtqocik
pyfghsvavjlqgpmejmsf
scbaqvofit
lkypjuqklahu
letbgqoipklkcpsu
dhbgdjyijmsiydlbtgnzgg
naxcetwmalxhcrccdzfqtlkyd
ejzfeyovxqtqocik
xpxobdwgnpaabesi
nprimartmqkmdxuy
pyfghsvavjlqgpmejmsf
ejzfeyovxqtqocik
lkypjuqklahu
lzyriieax
pgecwrltawtlsjq
rwwvmt
ltztrqdagsccmjvclmgabyalx
ypsynxykslwvktwrxez
twksexaxxpkiauklmyrzzxhn
pgecwrltawtlsjq
ejzfeyovxqtqocik
hjkkdltwjzkbeu
pyfghsvavjlqgpmejmsf
nprimartmqkmdxuy
pgecwrltawtlsjq
lkypjuqklahu
ejzfeyovxqtqocik
lzyriieax
pgecwrltawtlsjq
scbaqvofit
ugqlf
zvhoamnbyaqjlds
hjkkdltwjzkbeu
pyfghsvavjlqgpmejmsf
ejzfeyovxqtqocik
pgecwrltawtlsjq
fqbifcuhxixbyxurfbbqyyb
nprimartmqkmdxuy
zvhoamnbyaqjlds
lkypjuqklahu
pgecwrltawtlsjq
ejzfeyovxqtqocik
dpgdzszwqrqrxqtqfybuz
lzyriieax
ypsynxykslwvktwrxez
doxjanznswfrwjcwvvczoga
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
zvhoamnbyaqjlds
hjkkdltwjzkbeu
pgecwrltawtlsjq
nzzvfdbblnkpmhzskz
letbgqoipklkcpsu
ejzfeyovxqtqocik
twksexaxxpkiauklmyrzzxhn
pgecwrltawtlsjq
zvhoamnbyaqjlds
bibnfjpruwcju
nprimartmqkmdxuy
ktnnpk
lkypjuqklahu
ejzfeyovxqtqocik
cshosjmdvfnf
lzyriieax
pgecwrltawtlsjq
pyfghsvavjlqgpmejmsf
xqnfpccysyetkjbqt
zvhoamnbyaqjlds
hjkkdltwjzkbeu
aeaiukjemltsziwlyjdl
rwwvmt
scbaqvofit
pgecwrltawtlsjq
ejzfeyovxqtqocik
zvhoamnbyaqjlds
aeaiukjemltsziwlyjdl
cpshruduoizkzu
lkypjuqklahu
pgecwrltawtlsjq
ypsynxykslwvktwrxez
cshosjmdvfnf
nprimartmqkmdxuy
pyfghsvavjlqgpmejmsf
naxcetwmalxhcrccdzfqtlkyd
omlbfiuvfvfluq
ejzfeyovxqtqocik
zvhoamnbyaqjlds
hjkkdltwjzkbeu
lzyriieax
aeaiukjemltsziwlyjdl
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
zvhoamnbyaqjlds
aeaiukjemltsziwlyjdl
ioxgxyq
ksawzcehoejnob
ejzfeyovxqtqocik
vmpiqmwormx
twksexaxxpkiauklmyrzzxhn
pgecwrltawtlsjq
dhbgdjyijmsiydlbtgnzgg
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
zbkoamuhhgxkovzazh
lkypjuqklahu
xqnfpccysyetkjbqt
aeaiukjemltsziwlyjdl
zvhoamnbyaqjlds
xpxobdwgnpaabesi
hjkkdltwjzkbeu
pgecwrltawtlsjq
nprimartmqkmdxuy
zbkoamuhhgxkovzazh
omlbfiuvfvfluq
ejzfeyovxqtqocik
lzyriieax
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
pgecwrltawtlsjq
zvhoamnbyaqjlds
zbkoamuhhgxkovzazh
cshosjmdvfnf
ejzfeyovxqtqocik
pyfghsvavjlqgpmejmsf
aeaiukjemltsziwlyjdl
ypsynxykslwvktwrxez
scbaqvofit
lkypjuqklahu
ltztrqdagsccmjvclmgabyalx
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
letbgqoipklkcpsu
ktnnpk
zvhoamnbyaqjlds
hjkkdltwjzkbeu
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
omlbfiuvfvfluq
ejzfeyovxqtqocik
nprimartmqkmdxuy
lzyriieax
pgecwrltawtlsjq
bibnfjpruwcju
zbkoamuhhgxkovzazh
zvhoamnbyaqjlds
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
aeaiukjemltsziwlyjdl
xqnfpccysyetkjbqt
pgecwrltawtlsjq
doxjanznswfrwjcwvvczoga
zbkoamuhhgxkovzazh
ejzfeyovxqtqocik
rwwvmt
lkypjuqklahu
twksexaxxpkiauklmyrzzxhn
zvhoamnbyaqjlds
hjkkdltwjzkbeu
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
dpgdzszwqrqrxqtqfybuz
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
omlbfiuvfvfluq
ejzfeyovxqtqocik
lzyriieax
nprimartmqkmdxuy
zvhoamnbyaqjlds
aeaiukjemltsziwlyjdl
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
vmpiqmwormx
cpshruduoizkzu
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
ypsynxykslwvktwrxez
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
ejzfeyovxqtqocik
zvhoamnbyaqjlds
lkypjuqklahu
ugqlf
hjkkdltwjzkbeu
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
scbaqvofit
aeaiukjemltsziwlyjdl
fqbifcuhxixbyxurfbbqyyb
cshosjmdvfnf
pgecwrltawtlsjq
pyfghsvavjlqgpmejmsf
zbkoamuhhgxkovzazh
zvhoamnbyaqjlds
xqnfpccysyetkjbqt
omlbfiuvfvfluq
ejzfeyovxqtqocik
lzyriieax
nprimartmqkmdxuy
ioxgxyq
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
pgecwrltawtlsjq
zvhoamnbyaqjlds
hjkkdltwjzkbeu
zbkoamuhhgxkovzazh
lkypjuqklahu
ejzfeyovxqtqocik
nzzvfdbblnkpmhzskz
naxcetwmalxhcrccdzfqtlkyd
aeaiukjemltsziwlyjdl
ksawzcehoejnob
twksexaxxpkiauklmyrzzxhn
pgecwrltawtlsjq
ktnnpk
zbkoamuhhgxkovzazh
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
zvhoamnbyaqjlds
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
omlbfiuvfvfluq
ejzfeyovxqtqocik
letbgqoipklkcpsu
lzyriieax
pgecwrltawtlsjq
ypsynxykslwvktwrxez
nprimartmqkmdxuy
zbkoamuhhgxkovzazh
zvhoamnbyaqjlds
hjkkdltwjzkbeu
aeaiukjemltsziwlyjdl
lkypjuqklahu
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
ejzfeyovxqtqocik
bibnfjpruwcju
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
xqnfpccysyetkjbqt
zbkoamuhhgxkovzazh
zvhoamnbyaqjlds
aeaiukjemltsziwlyjdl
vmpiqmwormx
pgecwrltawtlsjq
rwwvmt
zbkoamuhhgxkovzazh
scbaqvofit
omlbfiuvfvfluq
ejzfeyovxqtqocik
lzyriieax
aeaiukjemltsziwlyjdl
dhbgdjyijmsiydlbtgnzgg
zvhoamnbyaqjlds
hjkkdltwjzkbeu
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
xpxobdwgnpaabesi
nprimartmqkmdxuy
lkypjuqklahu
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
ejzfeyovxqtqocik
pgecwrltawtlsjq
zvhoamnbyaqjlds
cpshruduoizkzu
zbkoamuhhgxkovzazh
twksexaxxpkiauklmyrzzxhn
doxjanznswfrwjcwvvczoga
aeaiukjemltsziwlyjdl
ypsynxykslwvktwrxez
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
omlbfiuvfvfluq
ejzfeyovxqtqocik
zvhoamnbyaqjlds
hjkkdltwjzkbeu
lzyriieax
cshosjmdvfnf
ltztrqdagsccmjvclmgabyalx
pyfghsvavjlqgpmejmsf
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
xqnfpccysyetkjbqt
lkypjuqklahu
pgecwrltawtlsjq
nprimartmqkmdxuy
zbkoamuhhgxkovzazh
zvhoamnbyaqjlds
ejzfeyovxqtqocik
dpgdzszwqrqrxqtqfybuz
aeaiukjemltsziwlyjdl
ktnnpk
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
ioxgxyq
zvhoamnbyaqjlds
hjkkdltwjzkbeu
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
cshosjmdvfnf
omlbfiuvfvfluq
ejzfeyovxqtqocik
pyfghsvavjlqgpmejmsf
lzyriieax
pgecwrltawtlsjq
scbaqvofit
zbkoamuhhgxkovzazh
lkypjuqklahu
zvhoamnbyaqjlds
aeaiukjemltsziwlyjdl
letbgqoipklkcpsu
nprimartmqkmdxuy
vmpiqmwormx
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
ejzfeyovxqtqocik
ypsynxykslwvktwrxez
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
zvhoamnbyaqjlds
cshosjmdvfnf
hjkkdltwjzkbeu
twksexaxxpkiauklmyrzzxhn
pgecwrltawtlsjq
pyfghsvavjlqgpmejmsf
xqnfpccysyetkjbqt
ksawzcehoejnob
bibnfjpruwcju
zbkoamuhhgxkovzazh
omlbfiuvfvfluq
ejzfeyovxqtqocik
lzyriieax
aeaiukjemltsziwlyjdl
rwwvmt
naxcetwmalxhcrccdzfqtlkyd
lkypjuqklahu
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
zvhoamnbyaqjlds
nprimartmqkmdxuy
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
ejzfeyovxqtqocik
pgecwrltawtlsjq
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
zvhoamnbyaqjlds
hjkkdltwjzkbeu
zbkoamuhhgxkovzazh
ugqlf
aeaiukjemltsziwlyjdl
pgecwrltawtlsjq
zbkoamuhhgxkovzazh
omlbfiuvfvfluq
ejzfeyovxqtqocik
cpshruduoizkzu
lzyriieax
zvhoamnbyaqjlds
fqbifcuhxixbyxurfbbqyyb
lkypjuqklahu
zbkoamuhhgxkovzazh
aeaiukjemltsziwlyjdl
pgecwrltawtlsjq
ypsynxykslwvktwrxez
scbaqvofit
cshosjmdvfnf
pyfghsvavjlqgpmejmsf
zbkoamuhhgxkovzazh
xqnfpccysyetkjbqt
ejzfeyovxqtqocik
nprimartmqkmdxuy
